changelog/README.md

```diff
@@ -2,6 +2,10 @@
 
 <!-- Note: contrary to 3.x, insert new entries *first* in their section -->
 
+### 4.14.2 (in progress)
+
+- [bug] JAVA-3002 JAVA-3005: Refresh entire node list when a new node is added
```

Comment:
we could also just reference one of these tickets and close the other one as a duplicate, let me know